.editorconfig

```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```

Comment:
any reason this isn't `indent_size = 2` ?

Reply:
Whoops! Thought I had opened the PR in the VHS org.

And, nah, it's more my default for readability.

I just released 1.2.0 on https://github.com/vhs/node-nfc-nci